components/canvas/raqote_backend.rs

```diff
@@ -185,7 +168,7 @@ pub struct SurfacePattern<'a> {
     image: raqote::Image<'a>,
     filter: raqote::FilterMode,
     extend: raqote::ExtendMode,
-    repeat: Repetition,
+    pub repeat: Repetition,
```

Comment:
I wasn't sure whether to make this field public or to write a getter. I'm open for an idiomatic suggestion.

Reply:
A getter seems reasonable to me.